### ISSUE TITLE
Phylo newick names

### ISSUE THREAD

**Issue by zafarali** [biopython/biopython#548]
Hi I have some newick data where the names of the clades are numbers. Is there a way for Phylo to make it recognize the numbers as strings so that it can be displayed?

For example:
Can I do
```
tree1 = Phylo.read(StringIO('((((19,8),18),17),(1,16),((((21,10),20),13),7));', 'newick')
Phylo.draw(tree1) # this draws a tree without labels
```
![image](https://cloud.githubusercontent.com/assets/6295292/7820729/04cb1f1e-03b8-11e5-89a0-b5342370b18e.png)

```
tree2 = Phylo.read(StringIO('(((("19","8"),"18"),"17"),("1","16"),(((("21","10"),"20"),"13"),"7"));'), 'newick')
Phylo.draw(tree2) # this draws a tree with labels surrounded by quotes
```
![image](https://cloud.githubusercontent.com/assets/6295292/7820737/0f01ffac-03b8-11e5-9519-c14c0ba50c94.png)

The second image shows what I would like to get but without quotes.

**Comment by etal** [comment]
Which version of Biopython are you using? On my machine, the integer labels are shown correctly using the first code example (no quotes needed).

**Comment by etal** [comment]
I believe this was fixed in September: 4df0acaf74879f3a791cf8b0d0161ac7c5056cd8

**Comment by zafarali** [comment]
I am using a work machine which hasn't been upgraded in a while and I am at 1.6.1
I shall upgrade and try again.
Thank you for your response!
